viz/cmd/uninstall.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/linkerd/linkerd2/pkg/k8s/resource"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newCmdUninstall() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "uninstall",
+		Args:  cobra.NoArgs,
+		Short: "Output Kubernetes resources to uninstall the Linkerd viz extension",
+		Long: `Output Kubernetes resources to uninstall the Linkerd viz extension.
```

Comment:
This suggestion avoids some discrepancy in naming:
```
$ linkerd viz
...
Available Commands:
  install     Output Kubernetes resources to install linkerd-viz extension
  uninstall   Output Kubernetes resources to uninstall the Linkerd viz extension
```

```suggestion
		Short: "Output Kubernetes resources to uninstall the linkerd-viz extension",
		Long: `Output Kubernetes resources to uninstall the linkerd-viz extension.
```